extensions/linear/src/components/IssueActions/IssueActions.tsx

```diff
@@ -421,6 +450,8 @@ export default function IssueActions({
           onChange={setDueDate}
         />
 
+        <Action.PickDate title="Set Reminder" shortcut={{ modifiers: ["cmd"], key: "h" }} onChange={setReminder} />
```

Comment:
All other actions use the `⇧` key for actions. Should we apply the same rules here?